src/lib/song.ts

```diff
@@ -70,7 +71,11 @@ export const loadNextRandomSongForPhase = async (target: 'day' | 'night', exclud
 
 	const song = getRandomSong(target, excludedSong);
 
-	await player?.load(createApiSongUrl(song, target));
+	try {
```

Comment:
this can be shortened into:
`await player?.load(createApiSongUrl(song, target)).catch(showErrorToast)`